Jenkinsfile

```diff
@@ -6,7 +6,7 @@ yproperties() // Sets releng approved global properties (SCM polling, build log
 CHANNELS = ['paasta']
 GIT_SERVER = 'git@github.com'
 PACKAGE_NAME = 'mirrors/Yelp/paasta'
-DIST = ['trusty', 'xenial', 'bionic']
```

Comment:
We still need trusty debs. Things have not changed from https://github.com/Yelp/paasta/pull/2342

Reply:
It seems that deadsnakes officially doesn't support Python 3.7 on Trusty: https://github.com/deadsnakes/issues/issues/63

Reply:
looks like some of the roles require paasta-tools because meshd depends on `paasta_dump_locally_running_services`. Maybe there's a way to break that dependency; maybe meshd doesn't always need it, or maybe we can publish `paasta_dump_locally_running_services` as a separate package that continues to run on py36 for trusty support.